docs/parameters.yaml

```diff
@@ -336,6 +336,27 @@ default: none
 components: ["origin"]
 ---
 
+############################
+#   Cache-level configs    #
+############################
+name: Cache.DataFileLocation
```

Comment:
It's not a "data file", is it?  It assumes that there's a directory -- how about `Cache.DataLocation`?